lib/src/diff/main.dart

```diff
@@ -296,21 +296,21 @@ List<Diff> diffBisect(String text1, String text2, double timeout,
   int k2end = 0;
   for (int d = 0; d < max_d; d++) {
     // Bail out if deadline is reached.
-    if ((new DateTime.now()).compareTo(deadline) == 1) {
+    if ((new DateTime.now()).compareTo(deadline!) == 1) {
```

Comment:
This could lead to a null exception. Because it is not sure that deadline is not null, this could be null. So maybe it should be:
    if (deadline != null && (new DateTime.now()).compareTo(deadline) == 1) {
      break;
    }